### ISSUE TITLE
In data prep, treat BCGNIS localities as real localities  with qualified names

### ISSUE THREAD

**Issue by mraross** [bcgov/ols-geocoder#191]
Instead of subLocality, represent BCGIS populated place names that aren’t DRA localities as fully qualified localities as follows:

IRs:
Boston Bar 8 near Boston Bar, BC

Unincorporated populated places:
Brentwood Bay in Central Saanich, BC

Duplicate unincorporated populated places should be qualified with their locality type as follows:
Tsawwassen (landing) in Delta, BC
Tsawwassen (community) in Delta, BC

When a user enters Boston Bar in an autocompletion box, the geocoder will return Boston Bar as the best choice since it is an exact string match, followed by the names of all Boston Bar Indian Reserves since they would be only partial string matches. No knowledge of frontGate or special keywords required!

This proposal should not require any changes to the online or batch geocoders; just data prep. This enhancement should eliminate confusion when a user is faced with bianyms in the autocompletion choice list. A bianym is a name that is used to identify more than one locality in the province (e.g., Mill Bay on Vancouver Island, Mill Bay in Northern BC).

**Comment by mraross** [comment]
All BCGNIS names should be qualified with their ITN localities, not just duplicates.

**Comment by gleeming** [comment]
FMW script logic has been updated to do the following to BCGNIS locality candidates:
1) restore cases so they are once again locality records
2) rename them all to include the descriptor "in" or "near" followed by the ITN locality name in which the BCGNIS point falls*
3) add an additional descriptive feature code in parentheses if disambiguation is required
4) leave locality mapping strings unchanged (so the BCGNIS entries won't include the additional descriptors)

*While testing, I noticed a few BCGNIS points spatially located within out of province ITN locality polygons, e.g. Tsawwassen Beach in Washington.  I added logic to drop these, resulting in 11 of the 3808 locality records being removed.
